README.md

```diff
@@ -1,2 +1,183 @@
 # AndcultureCode.CSharp.Sitefinity.Testing
+
 .NET Core wrapper objects representing built-in Sitefinity module models and associated service files to interact and write integration tests to communicate with the Sitefinity OData API.  This should be used to create integration tests for built-in module content types, custom fields in those built-in module content types, and any dynamic modules built on top of Sitefinity.
+
+NOTE: This is a rewrite of the Sitefinity CMS test framework found [here](https://github.com/Sitefinity/test-framework-core).  This repo was generated instead of using the Sitefinity repo for several reasons:
+
+1. The Sitefinity repo uses .NET Framework (not .NET Core)
+2. The Sitefinity repo was using less than desirable code for instance...
+   - dictionaries in content models
+   - no base test service
+   - content models had name of service contained with it
+3. The Sitefinity repo had problems with it
+   - until recently, tests for publishing were failing
+   - some of the tests were failing when asserting on successful creation of content
+4. The Sitefinity repo wasn't a comprehensive suite of tests covering all of their built-in content types
+
+This project is stand alone without any associated tests intentionally.  This is meant to serve as a base for generating your own project specific testing project.  
+
+## Setup 
+
+Before referencing this project, please ensure you have a Sitefinity instance readily available and an account set up in your Sitefinity instance.  See the **Installation** section found [here](https://github.com/Sitefinity/test-framework-core) for more details on the setup of the Sitefinity instance account.
+
+Copy the contents of `appsettings.json.sample` and paste them into your test project's `appsettings.json` file.  Update the settings to match th account for your Sitefinity instance.
+
+When generating your own project specific test suite using this as a dependency, you should consider having a *testing* environment configured which is available to run your tests against before continuing code changes through your CI/CD pipeline.
+
+## Example
+
+Below is an example of an integration test suite for the Blog Post Sitefinity service.
+
+```csharp
```

Comment:
Suggestion: One thing that we’ll have to keep in mind as the code changes is to update the sample code you attached here. Perhaps you could add a comment to the `BlogPostServiceTests` class indicating that the README needs to be updated if it changes.